ecommerce_integrations/fixtures/custom_field.json

```diff
@@ -0,0 +1,55 @@
+[
+ {
+  "allow_in_quick_entry": 0,
```

Comment:
Custom field in this app should be managed by settings doctype's setup_custom_field function

https://github.com/frappe/ecommerce_integrations/blob/5cafd9129edcd4951fff4a51117e6bc58c91cea6/ecommerce_integrations/unicommerce/doctype/unicommerce_settings/unicommerce_settings.py#L189

Reply:
This is required to not automatically add custom fields which might not be required by everyone.